gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -88,6 +94,9 @@
                 .map(Loadable::value)
                 .switchMap(this::getFeaturesStream)
                 .map(MapContainerViewModel::toMapPins));
+    this.mapLayerUpdateRequests =
```

Comment:
Can we simplify this by eliminating `mapLayerTypeClicks` and just making `mapLayerUpdateRequests` a `MutableLiveData` that gets a new Event on each click (`.setValue(Event.create(..))`)?

Reply:
Thanks! Done